modyn/protos/supervisor.proto

```diff
@@ -33,11 +33,12 @@ message PipelineStageDatasetMsg {
 }
 
 message PipelineStageCounterCreateParams {
-    int32 new_data_len = 1;
+    string title = 1;
+    optional int32 new_data_len = 2;
 }
 
 message PipelineStageCounterUpdateParams {
-    int32 batch_size = 1;
+    int32 increment = 1;
```

Comment:
this might be causing failures in the integration test :)

Reply:
yes, forgot copy the adjustments in the client from my other branch